CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
 # osquery Changelog
 
+<a name="5.4.0"></a>
+## [5.4.0](https://github.com/osquery/osquery/releases/tag/5.4.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.3.0...5.4.0)
+
+Representing commits from 15 contributors! Thank you all.
+
+### New Features
+
+- We're extending macOS Endpoint Security to include File Integrity monitoring. Check out the new `es_process_file_events` table. ([#7579](https://github.com/osquery/osquery/pull/7579))
```

Comment:
Suggest we word this as "Improved File Integrity Monitoring on macOS using EndpointSecurity with the addition of the new table `es_process_file_events`"

Where "improved" is "Faster" or "Improved performance" ... @sharvilshah might have a suggestion on what the primary user-benefit over OpenBSM is, besides the fact that OpenBSM is scheduled for deprecation

Reply:
We never implemented OpenBSM backed `process_file_events` table on macOS (partly because it's such a pain) -- the current `process_file_events` table is Linux/auditd only, so this ES backed `es_process_file_events` is brand new on macOS :)

Reply:
Ah, right! So the real new capability here is specifically that users can get process-context on file-events. On macOS, where previously they could only get those separately.